src/pages/Login/Validation.ts

```diff
@@ -0,0 +1,20 @@
+interface errorsValue {
+  emailError: string;
+  passwordError: string;
+}
+
+export const validation = (email: string, password: string) => {
+  const regex = /^([\w.%+-]+)@([\w-]+\.)+([\w]{2,})$/i;
```

Comment:
Add a comment on what precisely this regex is doing or delete it and create it in a way that all of your teammates will understand